packages/twenty-server/src/engine/integrations/environment/environment-variables.ts

```diff
@@ -151,6 +146,38 @@ export class EnvironmentVariables {
   @IsOptional()
   FRONT_AUTH_CALLBACK_URL: string;
 
+  @CastToBoolean()
+  @IsOptional()
+  @IsBoolean()
+  AUTH_PASSWORD_ENABLED: boolean = false;
```

Comment:
`true` default value is probably a better choice here

Reply:
I have made this change and pushed as well. Don't know if I was supposed to.